src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -50,6 +50,39 @@ limitations under the License.
       <arg name="Data"   type="char_string" optional="true"/>
     </command>
 
+    <command source="client" code="0x04" name="GetProgramGuide" response="ProgramGuideResponse" optional="true">
+      <description>This command retrieves the program guide. It accepts several filter parameters to return specific schedule and program information from a content app. The command shall receive in response a ProgramGuideResponse.</description>
+      <arg name="StartTime" type="epoch_s"/>
+      <arg name="EndTime" type="epoch_s"/>
+      <arg name="ChannelList" type="ChannelInfoStruct" array="true"/>
+      <arg name="PageToken" type="PageTokenStruct"/>
```

Comment:
Optional tags are missing here for optional parameters defined in this file.

Reply:
Updated.